apis/Google.Cloud.Bigtable.V2/Google.Cloud.Bigtable.V2/BigtableClient.cs

```diff
@@ -290,6 +352,22 @@ public sealed partial class BigtableClientImpl : BigtableClient
             return GetUnderlyingClient().MutateRowAsync(request, callSettings);
         }
 
+        /// <inheritdoc/>
+        public override Task<MutateRowResponse> MutateRowAsync(
+            MutateRowRequest request,
+            CancellationToken cancellationToken)
+        {
+            if (request.AppProfileId == null)
+            {
+                request.AppProfileId = _appProfileId;
+            }
+
+            return GetUnderlyingClient().MutateRowAsync(request, cancellationToken);
+        }
+
+        => MutateRowAsync(
```

Comment:
OK, not sure what's happened here.
Will look into it.

Reply:
This is from the Bigtable custom generator, not the normal generator. I'll look into this one.

Reply:
Ah right, thanks. That's a relief!